src/subscriptions.js

```diff
@@ -54,12 +54,12 @@ export default tracker => ({ dispatch }) => next => action => {
         'A startSubscription action needs a `get` function to load data'
       );
 
-      stopSubscription(action);
-
       const { key, subscribe } = action.payload;
       const subscription = subscribe();
       const { subscriptionId } = subscription;
 
+      stopSubscription(key);
```

Comment:
I think this should stay before `const subscription = subscribe();`, otherwise we are stopping the newly created subscription each time!

Reply:
Ah, yes! agree!